contrib/dropwizard/README.md

```diff
@@ -0,0 +1,84 @@
+# OpenCensus DropWizard Util for Java
+
+The *OpenCensus DropWizard Util for Java* provides an easy way to send metrics from your DropWizard
+project to OpenCensus.
+
+## Quickstart
```

Comment:
What do I need to add in the code to install this MetricProducer?

Reply:
Added the example code.

Reply:
Other way, add register method inside DropWizardMetrics to register MetricProducer. Something like this.

```
  public static void register(com.codahale.metrics.MetricRegistry metricRegistry) {
    MetricProducerManager metricProducerManager = Metrics.getExportComponent().getMetricProducerManager();
    metricProducerManager.add(new DropWizardMetrics(metricRegistry));
  }
```

any suggestion?